### DISCUSSION TITLE
Is it possible to have my own marketplace on PaaS solution?

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#146] [Q&A] [answered]
Hi guys,

Can you confirm that the markeplace is something at console level?
So, if I have a company on Mia-Platform Console PaaS (i.e. console.cloud.mia-platform.eu), I cannot have my own marketplace, right? In this case, can I make a template that is visible only in certain companies?

Thanks in advance.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @Lucascanna as of now the Marketplace is shared across the Console installation, therefore there is no way of having private marketplace without having a private installation; it is in our roadmap to evolve the marketplace for our SaaS Console in order to enable such use case

**Comment by Danielecina** [comment]
Hi Luca, Today is not supported, but it can be a good feature.
